Makefile

```diff
@@ -26,6 +26,8 @@ circle-setup:
 	cp -R ./* $(DRUPAL_SITE_PATH)/profiles/ding2/
 	# Install the site using the ding2 profile
 	cd $(DRUPAL_SITE_PATH) && drush site-install ding2 --db-url=mysql://ubuntu@127.0.0.1/circle_test -y
+	# Silence all notices and warnings prior to testing.
```

Comment:
Normally you would not want to do this. It would be nice with a word or two about *why* this is necessary. I cannot see anything about it in the commit message either.